specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/schedule.json

```diff
@@ -418,11 +418,18 @@
         "Day",
         "Hour",
         "Week",
-        "Month"
+        "Month",
+        "Minute"
       ],
       "x-ms-enum": {
         "name": "ScheduleFrequency",
-        "modelAsString": true
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Minute",
```

Comment:
This is broken and the cause of the build failures.  The values listed here must match the above enum values, see https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum

Reply:
Added all missing enum values